AISKU/package.json

```diff
@@ -22,10 +22,10 @@
     },
     "dependencies": {
         "@microsoft/applicationinsights-analytics-js": "^1.0.0-beta.12",
```

Comment:
can u change this to 1.0.0-beta.11. this should be 11 and not 12.

Reply:
I bumped it to the version that will eventually be published. There was a single change to the analytics plugin since it uses the instrumentation key field

Reply:
i see, if you updated this, not sure why I am not seeing a diff here.

Reply:
i changed to 11 and then i realized u had published 11 already from a different branch but version was behind.